pkg/sfu/downtrack.go

```diff
@@ -306,6 +306,9 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 	d.bound.Store(true)
 	d.bindLock.Unlock()
 
+	if d.onMaxLayerChanged != nil {
+		d.onMaxLayerChanged(d, d.MaxLayers().Spatial)
+	}
```

Comment:
`MediaTrackSubscriptions` was setting max layer to `HIGH` on adding a down track. But, that was hanging without much context. So, calling this from down track on bind so that it comes from the subscriber.